docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,145 @@
+# Getting started with supercomputing at CSC
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computing! This page provides guidance for getting started with
+using our HPC resources. It is recommended that new users complete the
+[CSC Computing Environment course](https://csc-training.github.io/csc-env-eff/),
+which provides a more in-depth introduction to using CSC services.
+
+## Available systems
+
+### Puhti
+
+New users are recommended to start working on the
+[Puhti supercomputer](../../computing/systems-puhti.md).
+Compared to Mahti, it has much more pre-installed software, more GPU nodes, and
+typically more available memory per CPU. Additionally, some Puhti CPU nodes have
+fast local NVMe storage, which is reserved only for GPU nodes on Mahti.
+
+### Mahti
+
+If you know that your computations are highly parallelizable, you should
+consider running them on the
+[Mahti supercomputer](../../computing/systems-mahti.md).
+Compared to Puhti, Mahti has many more CPU nodes and cores per node. Mahti is
+intended for computations that are able to effectively utilize at least an
+entire CPU node.
+
+Additionally, while Mahti has fewer GPU nodes than Puhti, the A100 GPUs on
+Mahti are considerably more powerful than the V100 GPUs on Puhti, which
+makes Mahti also suitable for machine learning applications.
+
+### LUMI
+
+LUMI is one of the fastest supercomputers in the world. It is intended primarily
+for running computations that benefit from the large amount of high-performance
+GPUs in its LUMI-G hardware partition. Whereas the GPUs on Puhti and Mahti are
+manufactured by Nvidia, the LUMI GPUs are made by AMD, so make sure that your
+GPU applications are able to use AMD drivers. LUMI has
+[its own documentation pages](https://docs.lumi-supercomputer.eu/).
+
+## Available interfaces
+
+### Web interface
+
+Puhti, Mahti and LUMI each have
+[their own web interface](../../computing/webinterface/index.md), which allows
+interacting with the supercomputer using a web browser. The web interface is a
+good choice for interactive computing, such as developing code or exploring
+and visualizing data. For this purpose, the web interface features multiple
+interactive applications, like
+[Visual Studio Code](../../computing/webinterface/vscode.md),
+[Jupyter](../../computing/webinterface/jupyter.md) and
+[RStudio](../../computing/webinterface/rstudio.md). For demanding computation,
+like running full-scale simulations or training neural networks, you will want
+to use the command-line interface, as it allows you to schedule your jobs and
+allocate more resources.
+
+### Command-line interface
+
+Effective high-performance computing is based on using the supercomputers'
```

Comment:
Not sure what is the best way to formulate this, but do you think some explanation/contrasting like this would make sense?
```suggestion
While the web interfaces are easy to use and a good starting point for using CSC's
supercomputers, the available capacity is limited as they are primarily intended for
interactive usage with low resource requirements. If you need to access more resources
(multiple CPU nodes or GPUs, for example), it is required to switch to using the
text-based Linux command-line interface (CLI) of the supercomputer. While this
may seem as an archaic way of working, it is truly powerful once you get a hang
of it.
```